packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -534,6 +558,14 @@ class BufferLogger extends Logger {
     )..start();
   }
 
+  @override
+  Status startSpinner({VoidCallback onFinish}) {
+    return SilentStatus(
```

Comment:
It looks like some Windows stdio supports ANSI escapes, do we need to drop the `terminal.supportsColor` check?
```
  /// On Windows, only versions of Windows 10 after v.1511
  /// ("TH2", OS build 10586) will be detected as supporting the output of
  /// ANSI escape sequences, and only versions after v.1607 ("Anniversary
  /// Update", OS build 14393) will be detected as supporting the input of
  /// ANSI escape sequences.
```

Reply:
we'd need to do a more complicated version check I think. If users use they new windows terminal it supports everything though, I'm not too worried about back porting our loggers to work on old pwsh exes

Reply:
Then should this be `AnsiSpinner` then?

Reply:
this is a buffer logger though, it only creates silent statuses

Reply:
Sigh.  I expanded the collapsed text too far and saw `WindowsStdoutLogger`.  Please ignore.

Reply:
No problem :)